src/test/scala/uk/gov/hmrc/mongo/MongoConnectorSpec.scala

```diff
@@ -10,9 +10,9 @@ import org.scalatest.{Matchers, WordSpec}
 class MongoConnectorSpec extends WordSpec with Matchers  {
 
   "MongoConnector" should {
-    "create a Mongo connection with the given options" in {
+    "create a Mongo connection with the given options" in  {
 
-      val connector = MongoConnector("mongodb://mongo-host:2000/mongo?connectTimeoutMS=1000&socketTimeoutMS=2000", failoverStrategy = None)
+      val connector = MongoConnector("mongodb://127.0.0.1:27017/test?connectTimeoutMS=1000&socketTimeoutMS=2000", failoverStrategy = None)
```

Comment:
Because I've changed the implementation to essentially make sure we have a connection established (I would even consider this a bug fix), I have to actually connect to mongo rather than just work round a fake connector that lazy loads the connection.